data/json/vehicle_parts.json

```diff
@@ -4000,6 +4000,23 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "beeper",
+        "name" : "back-up beeper",
+        "symbol" : "*",
+        "color" : "light_gray",
+        "broken_symbol" : "*",
+        "broken_color" : "dark_gray",
+        "damage_modifier" : 10,
+        "durability" : 90,
+        "bonus" : 28,
+        "item" : "beeper",
+        "difficulty" : 2,
+        "flags" : ["BEEPER", "ODDTURN"],
+        "breaks_into" : [
+            {"item": "steel_chunk", "min": 0, "max": 1}
```

Comment:
Some electronics would be nice here. Amplifiers sound like something to be expected from things that make sound.